auto_tune_vllm/utils/version_manager.py

```
@@ -0,0 +1,299 @@
+        Args:
+            version: Version string. If None, returns path to latest.
+        
+        Returns:
+            Path to the defaults file
+        """
+        if version is None:
+            return self.defaults_dir / "latest.yaml"
+        
+        version_info = self.get_version(version)
+        if version_info is None:
+            raise ValueError(f"vLLM defaults version {version} not found")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)